plugins(prometheus) add new metrics since 0.9.x (#2984)
Kong/docs.konghq.com: app/_hub/kong-inc/prometheus/index.md

  writing, and number of accepted connections.
- **Target Health**: The healthiness status (`healthchecks_off`, `healthy`, `unhealthy`, or `dns_error`) of Targets
  belonging to a given Upstream.

> - **Dataplane Status**: The last seen timestamp, config hash and config sync status for

Reviewer: ```suggestion
- **Dataplane Status**: The last seen timestamp, config hash, and config sync status for
```

> data plane nodes are exported on control plane.

Reviewer: ```suggestion
data plane nodes is exported to control plane.
```
